### ISSUE TITLE
Server Error (500) For integrating `Retire.js Scan`

### ISSUE THREAD

**Issue by chanduaki** [DefectDojo/django-DefectDojo#4713]
**Bug description**
when trying to upload the `Retire.js scan` results in the defect dojo , Getting `<h1>Server Error (500)</h1`

**Steps to reproduce**
1. Turnon the defectdojo `sudo docker-compose up -d`
2. scan the `retire.js` using `retire --jspath dvna --outputformat json --outputpath retire_report.json`
3. Trying to upload in 2 methods
    Method1: `CLI`
    ```
      curl -X POST "http://localhost:8080/api/v2/import-scan/" -H  "accept: application/json" -H "Authorization: Token 24****" -H  "Content-Type: multipart/form-data" -H  "X-CSRFToken: D1cByqi*********" -F "scan_date=2021-06-25" -F "minimum_severity=Info" -F "active=true" -F "verified=true" -F "scan_type=Retire.js Scan" -F "file=@retire_report.json;type=application/json" -F "engagement=8" -F "close_old_findings=false" -F "push_to_jira=false"
    ```
    Method2 : `API`
 4. Response got `<h1>Server Error (500)</h1`
 5. check the docker containers using 
 ```
 sudo docker-compose ps
              Name                            Command                State                                Ports                            
-------------------------------------------------------------------------------------------------------------------------------------------
django-defectdojo_celerybeat_1     /wait-for-it.sh mysql:3306 ...   Exit 73                                                                
django-defectdojo_celeryworker_1   /wait-for-it.sh mysql:3306 ...   Up                                                                     
django-defectdojo_initializer_1    /wait-for-it.sh mysql:3306 ...   Exit 0                                                                 
django-defectdojo_mysql_1          docker-entrypoint.sh mysql ...   Up        3306/tcp, 33060/tcp                                          
django-defectdojo_nginx_1          /entrypoint-nginx.sh             Up        80/tcp, 0.0.0.0:8080->8080/tcp, 0.0.0.0:8443->8443/tcp       
django-defectdojo_rabbitmq_1       docker-entrypoint.sh rabbi ...   Up        15691/tcp, 15692/tcp, 25672/tcp, 4369/tcp, 5671/tcp, 5672/tcp
django-defectdojo_uwsgi_1          /wait-for-it.sh mysql:3306 ...   Up     

```
 6. Check the container logs
 ```
 uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "./dojo/api_v2/serializers.py", line 1097, in save
uwsgi_1         |     parser_findings = parser.get_findings(data.get('file', None), test)
uwsgi_1         |   File "./dojo/tools/retirejs/parser.py", line 19, in get_findings
uwsgi_1         |     tree = self.parse_json(json_output)
uwsgi_1         | AttributeError: 'RetireJsParser' object has no attribute 'parse_json'
uwsgi_1         | 
uwsgi_1         | During handling of the above exception, another exception occurred:
uwsgi_1         | 

```

**Expected behavior**
- Upload the scan results to defect dojo dashboard for my given engagement

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [x] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Unbuntu20.04

**Comment by damiencarol** [comment]
@chanduaki which version are you using?

**Comment by vijaytejak** [comment]
@damiencarol 1.14 Version

**Comment by chanduaki** [comment]
1.14 version @damiencarol

**Comment by damiencarol** [comment]
This issue is fixed in `1.15.0`.
